shared/js/ui/models/feedback-form.es6.js

```diff
@@ -0,0 +1,55 @@
+const Parent = window.DDG.base.Model
+
+function FeedbackForm (attrs) {
+    attrs = attrs || {}
+    attrs.isBrokenSite = attrs.isBrokenSite || false
+    attrs.url = attrs.url || ''
+    attrs.message = attrs.message || ''
+    attrs.canSubmit = false
+    attrs.submitted = false
+
+    attrs.browser = attrs.browser || ''
+    attrs.browserVersion = attrs.browserVersion || ''
+    attrs.extensionVersion = attrs.extensionVersion || ''
+
+    Parent.call(this, attrs)
+
+    this.updateCanSubmit()
+}
+
+FeedbackForm.prototype = window.$.extend({},
+    Parent.prototype,
+    {
+        modelName: 'feedbackForm',
+
+        submit: function () {
+            if (!this.canSubmit) { return }
+
+            console.log('SUBMIT!!!')
```

Comment:
WIP?

Reply:
Yep this is WIP

Reply:
Yeah, just so we can get the bulk of the review out of the way for today.